apps/ui/src/content/changelog/2025-11-07-kimi-k2-thinking-model.md

```
@@ -0,0 +1,40 @@
+image:
+  src: "/changelog/kimi-k2-thinking.png"
+  alt: "Kimi K2 Thinking model now available on LLM Gateway"
+  width: 1768
+  height: 677
+---
+
+We're excited to announce support for **Kimi K2 Thinking**, Moonshot AI's advanced reasoning model, now available through LLM Gateway. Experience powerful thinking capabilities with extensive context and efficient prompt caching.
+
+## 📊 Model Specifications
+
+**Kimi K2 Thinking**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)